model/contract/Action.ttl

```diff
@@ -35,3 +40,25 @@ ids:includedIn
     rdfs:range ids:Action;
     rdfs:comment "The subject transitively asserts the object Action."@en ;
 .
+
+ids:pxpEndpoint
+     a owl:ObjectProperty;
+     rdfs:label "has PXP endpoint"@en;
+     rdfs:domain ids:Action;
+     rdfs:range ids:PxpEndpoint ;
+     rdfs:comment "The reference to the endpoint which operates a specific action, URI or reference to an associated PXP interface."@en
+.
+
+ids:pxpInterfaceDescription a owl:ObjectProperty;
+     rdfs:label "pxp interface description"@en;
+     rdfs:domain ids:PxpEndpoint;
+     rdfs:range  xsd:anyURI;
+     rdfs:comment "The reference to a URI that prodived the interface description of a PXP."@en;
+.
+
+ids:pxpAccessURI a owl:ObjectProperty;
+     rdfs:label "pxp access URI"@en;
+     rdfs:domain ids:PxpEndpoint;
+     rdfs:range  xsd:anyURI;
+     rdfs:comment "The reference to a URI that expresses an exact endpoint of that PXP."@en;
```

Comment:
Do we really mean "an" exact endpoint, or "the" exact endpoint?

Reply:
"an" endpoint because there might be more that one implementation for a PIP interface. (Different information sources be available)